### DISCUSSION TITLE
Windows 8.1?

### DISCUSSION THREAD

**Discussion by jjensen6823** [nefarius/HidHide#67] [Q&A] [answered]
Is there a version of HidHide that is compatible with Windows 8.1 (or 7)?  When I tried installing it on 7, I thought it gave me a message box that it was only for 8.1 or newer, but it looks like the versions on the download tab are all for 10/11?  And I was not able to get HidGuardian to work in 7 because of driver signature.  Very frustrating.

**Comment by nefarius** [comment, marked as answer]
As stated literally everywhere on this project, no, nothing below Windows 10 is supported. Please forward your frustration to Microsoft who chose to not back port new kernel APIs to 8.1 and 7. That's life.

From a kernel developer perspective anything below Windows 10 is dead and not worth developing for. Be my guest to port HidHide to Windows 7 yourself, it's open source after all.

**Reply by Kanuan** [reply]
Actually on the project's READM it's written:

```
## Package content

HidHide is a kernel-mode filter driver available for Windows 8.1 or higher (KMDF 1.13+). (...)
```

I'd make a PR to change this but I checked and 1.13 is actually Win8 compatible, so I'm not sure if you are confusing this with another project that is Win10 only, like dshidmini.

**Reply by nefarius** [reply]
Heh, yeah because this is *technically* correct but since we can not **sign** for Win 8.x without WHQL procedure that is a showstopper. PR welcome 👍
